setup.py

```diff
@@ -128,5 +128,8 @@ def read(fname):
     download_url='%s/v%s.tar.gz' % (url, version),
     keywords=['OMERO.CLI', 'plugin'],
     cmdclass={'test': PyTest},
-    tests_require=['pytest', 'restview', 'mox3'],
+    tests_require=[
+        'omero-py>=5.17.0',
```

Comment:
Today's release 5.18.0 as Python 3.8 as the min. We could require ``omero-py>=5.18.0``

Reply:
```suggestion
        'omero-py>=5.18.0',
```